bbb-graphql-actions/src/actions/meetingLayoutSetScreenshareAsContent.ts

```
@@ -0,0 +1,30 @@
+export default function buildRedisMessage(sessionVariables: Record<string, unknown>, input: Record<string, unknown>): RedisMessage {
+  throwErrorIfNotPresenter(sessionVariables);
+  throwErrorIfInvalidInput(input,
+      [
+        {name: 'screenshareAsContent', type: 'boolean', required: true},
+      ]
+  )
+
+  const eventName = 'SetScreenshareAsContentReqMsg';
+
+  const routing = {
+    meetingId: sessionVariables['x-hasura-meetingid'] as String,
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)